### PR TITLE
remove negation from stop words

### PR DESCRIPTION
**Pull request by chejennifer**
svindex differ: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/chejennifer_base_uae_mem_2024_07_09_20_03_18.html

base has all the stop words
test has the negation stop words removed

### CODE REVIEW

**Comment by pradh** [comment]
This looks good!  For diff #1, it seems hyphenating or separating noninstitutionalized really helps:

- https://screenshot.googleplex.com/9ZEF7nTt4ukkEro
- https://screenshot.googleplex.com/BvoZDc2jUfQnB9y

Wanna try?  Or defer?

**Review by pradh — APPROVED** [review]
If you decide to defer, file a bug or sneak it into another fix.

Generally, its worth revisiting "nonX" words in the description for whether "non-X" or "non X" is better.

**Comment by chejennifer** [comment]
> This looks good! For diff #1, it seems hyphenating or separating noninstitutionalized really helps:
> 
> * https://screenshot.googleplex.com/9ZEF7nTt4ukkEro
> * https://screenshot.googleplex.com/BvoZDc2jUfQnB9y
> 
> Wanna try? Or defer?

diffs for updated index with non-institutionalized & there seem to be some new golden diffs as well: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/chejennifer_base_uae_mem_2024_07_10_12_11_30.html
- diff #5 feels like a bit of a loss because "non institutionalized" is showing up above "not in labor force" & the query asked for "unemployment"
- golden diffs seem ok

**Comment by pradh** [comment]
> diffs for updated index with non-institutionalized & there seem to be some new golden diffs as well: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/chejennifer_base_uae_mem_2024_07_10_12_11_30.html

Thanks!
 
> * diff 5 feels like a bit of a loss because "non institutionalized" is showing up above "not in labor force" & the query asked for "unemployment"

I feel not too bad because "not in labor force" was ahead of that.  I think we should really fix the typo in the query -- https://screenshot.googleplex.com/3ACn93LitMfrubD (maybe have a bug for that?)

On diff #2, it still seems odd, I think we might want to be consistent - https://screenshot.googleplex.com/6wXTzwkhzvDYtiS.  Keep "civilians" in all those variables or drop them, consistently.  Feel free to defer though.

> * golden diffs seem ok

Agreed!

**Review by pradh — APPROVED** [review]

**Comment by chejennifer** [comment]
> > diffs for updated index with non-institutionalized & there seem to be some new golden diffs as well: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/chejennifer_base_uae_mem_2024_07_10_12_11_30.html
> 
> Thanks!
> 
> > * diff 5 feels like a bit of a loss because "non institutionalized" is showing up above "not in labor force" & the query asked for "unemployment"
> 
> I feel not too bad because "not in labor force" was ahead of that. I think we should really fix the typo in the query -- https://screenshot.googleplex.com/3ACn93LitMfrubD (maybe have a bug for that?)
> 
> On diff #2, it still seems odd, I think we might want to be consistent - https://screenshot.googleplex.com/6wXTzwkhzvDYtiS. Keep "civilians" in all those variables or drop them, consistently. Feel free to defer though.
> 
> > * golden diffs seem ok
> 
> Agreed!

Updated to keep "civilians" in all variables and diffs look ok to me: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/chejennifer_base_uae_mem_2024_07_10_13_44_47.html

& updated the query typo since I'm here already

**Review by pradh — APPROVED** [review]
Thank you!

File: tools/nl/embeddings/input/base/sheets_svs.csv [hunk 1]
```diff
@@ -2089,27 +2089,27 @@ Count_Person_BlackOrAfricanAmericanAlone_ResidesInNursingFacilities,Number of Bl
...
-Count_Person_Civilian_NonInstitutionalized_ForeignBorn,number of NonInstitutionalized Foreign Born Civilian
-Count_Person_Civilian_NonInstitutionalized_HispanicOrLatino,number of NonInstitutionalized Hispanic or Latino Civilian
-Count_Person_Civilian_NonInstitutionalized_Native,number of NonInstitutionalized native Civilian
-Count_Person_Civilian_NonInstitutionalized_NativeHawaiianOrOtherPacificIslanderAlone,number of NonInstitutionalized Native Hawaiian or Other Pacific Islander Civilian
-Count_Person_Civilian_NonInstitutionalized_OneRace,number of NonInstitutionalized single race Civilian
-Count_Person_Civilian_NonInstitutionalized_PovertyStatusDetermined,number of NonInstitutionalized Civilian with poverty status determined
-Count_Person_Civilian_NonInstitutionalized_ResidesInHousehold,number of NonInstitutionalized Civilian who resides in household
-Count_Person_Civilian_NonInstitutionalized_TwoOrMoreRaces,number of NonInstitutionalized muti-race Civilian
-Count_Person_Civilian_NonInstitutionalized_WhiteAlone,number of NonInstitutionalized white Civilian
+Count_Person_Civilian_FamilyHousehold_NonInstitutionalized,Number of civilian in family household
```
Comment (pradh) [inline tools/nl/embeddings/input/base/sheets_svs.csv:2092]: plural - civilians

Comment (chejennifer) [reply tools/nl/embeddings/input/base/sheets_svs.csv]: done

svindex diffs: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/chejennifer_base_uae_mem_2024_07_10_16_28_40.html
- no new diffs that are concerning

**Comment by chejennifer** [comment]
thanks for the reviews!

**Pull request merged**